File: pset1/mario/mario.c
#include<cs50.h>
#include<stdio.h>

int main(void){


    //asking user for input of height between 1 and 8 inclusive
    int x;
    do{
        x = get_int("Height: ");

    }while(x<1 || x>8);

    //iterate through the ith row 
    for(int i=0;i<x;i++){
        // form a left aligned pyramid of " "
        for(int j=x-1;j>i;j--){
            printf(" ");
        }
        // form a left aligned pyramid of #
        for(int k=-1;k<i;k++){
            printf("#");
        }
        //prints the gap between the two pyramids
        printf("  ");
        //forms a righ aligned pyramid of #
        for(int k=-1;k<i;k++){
            printf("#");
        }
        //prints the each row on a new line
        printf("\n");
    }
}

File: pset6/mario.py
from cs50 import get_int

def main():
    while True:
        x = get_int("Height: ")
        if x>=1 and x<=8:
            break
    for i in range(x):
        for j in range(x):
            if i+j>=x-1:
                print("#", end="")
            else:
                print(" ", end="")
        print()


if __name__ == "__main__":
    main()



